bridge/client/app/_components/ktb-copy-to-clipboard/ktb-copy-to-clipboard.component.scss

```diff
@@ -16,7 +16,7 @@
   }
 }
 
-.dt-copy-to-clipboard-btn-button {
+.dt-copy-to-clipboard-btn-button, .dt-copy-to-clipboard-successful {
   min-width: 10px !important;
   padding: 0 6px;
```

Comment:
May I ask you to change the horizontal `padding` to `5px`, so that the two buttons (copy and reveal value) have the same width?

![image](https://user-images.githubusercontent.com/6098219/165294376-67542a4f-5ff1-4c21-9d16-008920293895.png)

![image](https://user-images.githubusercontent.com/6098219/165294458-8367a905-6c78-47c8-af73-9669b25b217d.png)


```suggestion
  padding: 0 5px;
```